### PR TITLE
Added Buttons

### PR DESCRIPTION
**Pull request by hussiantabha**
Buttons Primary
Buttons Outline
Buttons Icons
Floating Buttons

### CODE REVIEW

File: docs.html [hunk 1]
```diff
@@ -98,7 +98,80 @@ <h2>Blog Title</h2>
...
+                    style="width: 80%; height: 298px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+          </div>
+          <div class="display-code-content">
+              <h2>Buttons with Icons</h2>
+              <div class="btn-container">
+                  <button class="btn btn-primary btn-icon">
+                    <svg class="icon" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M10 18V16H8V14H10V12H12V14H14V16H12V18H10Z" fill="currentColor" /><path fill-rule="evenodd" clip-rule="evenodd" d="M6 2C4.34315 2 3 3.34315 3 5V19C3 20.6569 4.34315 22 6 22H18C19.6569 22 21 20.6569 21 19V9C21 5.13401 17.866 2 14 2H6ZM6 4H13V9H19V19C19 19.5523 18.5523 20 18 20H6C5.44772 20 5 19.5523 5 19V5C5 4.44772 5.44772 4 6 4ZM15 4.10002C16.6113 4.4271 17.9413 5.52906 18.584 7H15V4.10002Z" fill="currentColor" /></svg>
```
Comment (munjalSaksham71) [inline docs.html:142]: Just a suggestion: Use Font-awesome Icons here.

File: docs.html [hunk 2]
```diff
@@ -98,7 +98,80 @@ <h2>Blog Title</h2>
...
+                    Upload File
+                  </button>
+                  <button class="btn btn-primary-outline btn-icon">
+                    <svg class="icon" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M8 11C7.44772 11 7 11.4477 7 12C7 12.5523 7.44772 13 8 13H15.9595C16.5118 13 16.9595 12.5523 16.9595 12C16.9595 11.4477 16.5118 11 15.9595 11H8Z" fill="currentColor" /><path d="M8.04053 15.0665C7.48824 15.0665 7.04053 15.5142 7.04053 16.0665C7.04053 16.6188 7.48824 17.0665 8.04053 17.0665H16C16.5523 17.0665 17 16.6188 17 16.0665C17 15.5142 16.5523 15.0665 16 15.0665H8.04053Z" fill="currentColor" /><path fill-rule="evenodd" clip-rule="evenodd" d="M5 3C3.89543 3 3 3.89543 3 5V19C3 20.1046 3.89543 21 5 21H19C20.1046 21 21 20.1046 21 19V5C21 3.89543 20.1046 3 19 3H5ZM7 5H5L5 19H19V5H17V6C17 7.65685 15.6569 9 14 9H10C8.34315 9 7 7.65685 7 6V5ZM9 5V6C9 6.55228 9.44772 7 10 7H14C14.5523 7 15 6.55228 15 6V5H9Z" fill="currentColor" /></svg>
```
Comment (munjalSaksham71) [inline docs.html:146]: Same: use Font-awesome icons

**Review by munjalSaksham71 — COMMENTED** [review]
Added Some Comments

File: docs.html (hunk 1 above)
Comment (hussiantabha) [inline docs.html:142]: I am more use to using SVG from next I will use Font Awesome Icons

Comment (munjalSaksham71) [reply docs.html:142]: Alright. Then its fine

**Review by munjalSaksham71 — APPROVED** [review]

File: app.css [hunk 3]
```diff
@@ -24,6 +26,12 @@ a{
 :root{
     --primary: blueviolet;
     --white: #fff;
+    --secondary: #155d9c;
+    --success: #198754;
+    --danger: #DC3545;
+    --warning: #FFC107;
+    --info: #0DCAF0;
+    --dark: #212529;
```
Comment (harshitadaswani) [inline app.css:34]: You have already defined it in components/button.css which you have imported into this file. So these lines are redundant and you can remove them

**Review by harshitadaswani — APPROVED** [review]
Just a suggestion: to add cursor: pointer property to buttons

File: app.css (hunk 3 above)
Comment (hussiantabha) [reply app.css:34]: Yeah

**Pull request merged**